lib/pages/people/view/people_page.dart

```diff
@@ -42,6 +42,7 @@ class _PeoplePageState extends State<PeoplePage> {
           onPressed: () {
             setState(() => searchClosed = !searchClosed);
           },
+          tooltip: 'Search'
```

Comment:
All strings should be localized. See the internationalization section in the documentation:
https://github.com/student-hub/acs-upb-mobile/blob/master/CONTRIBUTING.md#internationalization

Reply:
fc8b2b706a6833e30a7bb8c7ab7cafd24abce559